apps/migrator/components/migratorEditor.tsx

```diff
@@ -121,10 +121,10 @@ const MigrateEditor = (): ReactElement => {
       </div>
       <div className="flex flex-wrap-reverse justify-center md:justify-between items-center p-4 md:py-0 gap-y-4">
         <div className="text-center md:text-left">
-          <h4 className="font-bold">Want to dig deeper?</h4>
+          <h2 className="font-bold">Want to dig deeper?</h2>
```

Comment:
Is the `h2` too big for this space?

Reply:
The h2 update was by way of the a11y check which spotted that the markup had "h" tags but not in sequential order, which will confuse screen readers. The tag isn't used for font size here, just more logical flow down the tree.